streaming/tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+  "extends": "../tsconfig.json",
+  "compilerOptions": {
+    "target": "esnext",
+    "module": "CommonJS",
+    "moduleResolution": "node",
+    "noUnusedParameters": false,
```

Comment:
I would prefer to prefix unused parameters with `_` rather than setting this. IIRC Typescript should not complain about them, as the `_` prefix explicitly marks them as non-used

Reply:
Yeah, this is more, I didn't want to change all the method signatures at once, so allowing this for now is the better route.

Reply:
But are there functions with unused parameters at the moment?

I removed this and could not find any